ecs-init/docker/docker_test.go

```diff
@@ -253,6 +253,14 @@ func validateCommonCreateContainerOptions(opts godocker.CreateContainerOptions,
 	if hostCfg.NetworkMode != networkMode {
 		t.Errorf("Expected network mode to be %s, got %s", networkMode, hostCfg.NetworkMode)
 	}
+
+	if len(hostCfg.CapAdd) != 1 {
```

Comment:
We can but none of the other tests in the file employ the assert package. 
We haven't even vendored this package for `ecs-init`. 

It though it would be strange to have two styles in the same file.